core/chains/evm/log/broadcaster.go

```diff
@@ -497,7 +496,7 @@ func (b *broadcaster) onReplayRequest(replayReq replayRequest) {
 
 		// Use a longer timeout in the event that a very large amount of logs need to be marked
 		// as consumed.
-		err := b.orm.MarkBroadcastsUnconsumed(replayReq.fromBlock, pg.WithParentCtx(ctx), pg.WithLongQueryTimeout())
```

Comment:
We need to be very careful with `WithLongQueryTimeout` uses. They need to use `WithoutDefaultTimeout` to be able to set a longer timeout.

Reply:
Added `WithoutDefaultTimeout `. Should I set a new longer timeout here as well?

Reply:
Leaving it up to the caller seems fine, but maybe do a quick check that they are doing something reasonable?